code/datums/mood_events/generic_positive_events.dm

```diff
@@ -239,3 +239,8 @@
 	description = "Fishing is relaxing"
 	mood_change = 5
 	timeout = 3 MINUTES
+
+/datum/mood_event/joywire
+	description = "<span class='boldnicegreen'>I feel so joyous! Oh, so joyous! WOOOOOOOOOOOOOOOOOOOOOO!!</span>\n"
```

Comment:
```suggestion
	description = span_boldnicegreen("I feel so joyous! Oh, so joyous!\n")
```

Less is more I think, still not exactly pleased with this but I'm open to suggestion